bitnami/mysql/values.yaml

```diff
@@ -28,7 +28,6 @@ image:
 
 service:
   ## Kubernetes service type
-  # nodePort:
```

Comment:
Not sure about this one, it doesn't appear in the production one and it is not documented cc @tompizmor

Reply:
Is it used in the service? Maybe we forgot to document it..

Reply:
No, it is not used:
```
$ ag 'nodePort' bitnami/mysql
bitnami/mysql/values.yaml
31:  # nodePort:
```

Reply:
then we should delete it from the values.yaml